lib/component-data/model.js

```diff
@@ -14,7 +14,7 @@ const MODEL_SAVE_TIMEOUT = 8000, // timeout for model.js pre-save hook
 export function save(uri, data) {
   const model = getModel(uri);
 
-  if (_.isFunction(model.save)) {
+  if (_.isObject(model) && _.isFunction(model.save)) {
```

Comment:
I'm curious about why you changed this?

Reply:
`model` will be undefined if a component doesn't have a `model.js`, but I hadn't caught it before (since beforehand we'd assume it had a server.js, or otherwise required server-side checks for logic)